dist/s_comment.py

```diff
@@ -89,6 +95,10 @@
         function_desc = False
     elif multiline:
         comment += line
+        # We want to preserve newlines for block comments that have multiple paragraphs.
+        if len(sline) == 1 and sline.startswith('*'):
```

Comment:
Does it actually equal `sline = '*'`?

Reply:
Ha! Good point.
I rebased that change into the original script commit.